GlotPress/GlotPress-WP
# Local: Expose multiple matching originals

## What?
Exposes that multiple originals match and enables their translation.

## Why?
Sometimes the same translation applies to different originals. Previously this case was handled by just using the first original.

## How?
With the exposure of all translatable strings in #1599, we can now use this box to show all other matching originals for easier editing. This also adds an OR operator (`||`) to the search box.

## Testing Instructions
1. Find a translation that is the same for different originals and rightclick it.
2. In the popover, the text "Multiple originals match, [show them](#)" should appear.
3. When you click it, the search strings box should show with this and other originals in view.

--
1. Find another translation or orignal that doesn't have multiple matching originals
2. The text should not show.

## Screenshots or screencast 
![multiple-matching-originals](https://user-images.githubusercontent.com/203408/230593553-706cfd4a-5f05-4e52-9079-5871c937c80d.gif)
